ui/app/utils/default_jobs/hello-world.js

```diff
@@ -49,19 +49,19 @@ export default `job "hello-world" {
 
       template {
         data        = <<EOF
-                        <h1>Hello, Nomad!</h1>
-                        <ul>
-                          <li>Task: {{env "NOMAD_TASK_NAME"}}</li>
-                          <li>Group: {{env "NOMAD_GROUP_NAME"}}</li>
-                          <li>Job: {{env "NOMAD_JOB_NAME"}}</li>
-                          <li>Metadata value for foo: {{env "NOMAD_META_foo"}}</li>
-                          <li>Currently running on port: {{env "NOMAD_PORT_www"}}</li>
-                        </ul>
+<h1>Hello, Nomad!</h1>
+<ul>
+  <li>Task: {{env "NOMAD_TASK_NAME"}}</li>
+  <li>Group: {{env "NOMAD_GROUP_NAME"}}</li>
+  <li>Job: {{env "NOMAD_JOB_NAME"}}</li>
+  <li>Metadata value for foo: {{env "NOMAD_META_foo"}}</li>
+  <li>Currently running on port: {{env "NOMAD_PORT_www"}}</li>
+</ul>
                       EOF
```

Comment:
You might be able to retain the spec as written using `<<-EOF` instead of `<<EOF`